src/System.Private.CoreLib/src/Internal/Runtime/InteropServices/WindowsRuntime/ActivationFactoryLoader.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Runtime.Loader;
+
+//
+// Types in this file marked as 'public' are done so only to aid in
```

Comment:
The tests for this will require Reg-Free WinRT, which isn't available on any of the Helix machines AFAIK. I'll make the type internal and add it to the ILLink.Trim.xml file to make sure it doesn't get trimmed out.